examples/mvdream/models/mvdream.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any, Dict, List, Union
+
+import numpy as np
+import torch
+
+from mvdream.camera_utils import get_camera
+from mvdream.ldm.models.diffusion.ddim import DDIMSampler
+from mvdream.model_zoo import build_model
+
+class MVDreamModel:
+    def __init__(self, model_name: str = "sd-v2.1-base-4view", dtype: str = "float16"):
+
+        if torch.cuda.is_available():
+            self.device_str = "cuda"
+        else:
+            self.device_str = "cpu"
+        self.torch_dtype = getattr(torch, dtype)
+
+        self.model = build_model(model_name)
+        self.model.device = self.device_str
+        self.model.to(self.device_str)
+        self.model.eval()
+
+        self.sampler = DDIMSampler(self.model)
+
+        # output 4 view with fixed angles
+        self.nviews = 4
+        self.camera = get_camera(self.nviews).to(self.device_str)
+
+    def __call__(
+        self,
+        prompts: Union[str, List[str]],
+        negative_prompts: Union[str, List[str]] = [""],       
+        num_inference_steps: int = 50,
+        guidance_scale: float = 7.5,
+        ddim_eta:float = 0.0,
+        image_size: int = 256,
+        seed: int = -1,
+    ) -> List[Dict[str, Any]]:
```

Comment:
Is the output a List of dictionaries (one for each prompt), where `key` is the view name, and `value` is the image?

Reply:
Was a bug :man_facepalming:, changed to `List[Image.Image]` now.